packages/devtools_app/lib/devtools.dart

```diff
@@ -5,6 +5,4 @@
 /// The DevTools application version
 // This version should only be updated by running tools/update_version.dart
 // that updates all versions for DevTools.
-// Note: a regexp in tools/update_version.dart matches the following line so
-// if you change it you must also modify tools/update_version.dart.
```

Comment:
please don't remove this note, as it is still relevant

Reply:
Removed from this PR. But it is very confusing. We need to fox it.

Reply:
Why is this confusing? We look for this line in the update_version.dart script, as the note says: https://github.com/flutter/devtools/blob/master/tool/update_version.dart#L186